src/components/ProjectForm/ProjectForm.jsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * Components Material UI
+ */
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+/**
+ * Hook for handle form
+ */
+import { useFormik } from "formik";
+
+import { projectFormValidations } from "./ProjectFormValidations";
+import "./ProjectForm.scss";
+
+function ProjectForm() {
+  const formik = useFormik({
+    initialValues: {
+      name: "",
+      contact: "",
+      description: "",
+    },
+    validate: projectFormValidations,
+    // --> Waiting for API
+    onSubmit: (values) => {
+      alert(JSON.stringify(values, null, 2));
+    },
+  });
+  const hasErrorName = !!formik.touched.name && !!formik.errors.name;
+  const hasErrorContact = !!formik.touched.contact && !!formik.errors.contact;
+  const hasErrorDescription =
+    !!formik.touched.description && !!formik.errors.description;
+
+  return (
+    <form
+      className="container-form"
+      onSubmit={formik.handleSubmit}
+      noValidate
+      autoComplete="off"
+    >
+      <h1 className="container-form__title">Create Project</h1>
+      <p>Make your project know and hire the best resumes for it.</p>
+
+      <div className="u-mb-1">
+        <TextField
+          id="name"
+          label="Name"
+          name="name"
+          type="text"
+          variant="outlined"
+          error={hasErrorName}
+          helperText={hasErrorName ? formik.errors.name : ""}
+          onChange={formik.handleChange}
+          onBlur={formik.handleBlur}
+          value={formik.values.name}
```

Comment:
can you order all this props alphabetically please? here and in all components you're passing props